CRM/PivotData/AbstractData.php

```diff
@@ -177,6 +187,51 @@ protected function replaceCustomLabels($label) {
     return $label;
   }
 
+  /**
+   * Loads data for the given custom field.
+   *
+   * @param array $field
+   *   Array with the field's data, with the field's name and the field's group
+   *
+   * @return array
+   */
+  protected function loadCustomFieldData($field) {
```

Comment:
since you only use the field name and group,  passing them as two arguments is better